README.md

```diff
@@ -154,6 +154,14 @@ Simple bootstrap project created by *quarkus-maven-plugin*
 This module covers some basic scenarios around reactive routes in general and also:
 - Validation on request params, request body and responses.
 
+### `vertx-web-client`
```

Comment:
```suggestion
### `http/vertx-web-client`
```

Reply:
jaxrs, reactive-routes and servlet-undertow in the same folder do not have this suffix. I an afraid, suggested change will just create more confusion.

Reply:
Well spotted! Pls, adjust these modules: jaxrs, reactive-routes and servlet-undertow too.

Reply:
What about http-{minimum/static/advanced}? Should they too become http/http-minimum and so on? It looks like docs for the folder never used this prefix[1]. Maybe we should change this behaviour, but let us do it in a separate task.

[1] Your initial commit for README: https://github.com/quarkus-qe/quarkus-test-suite/commit/62cf97342b3f7209b316222b4120ff6a93270276

Reply:
Fine with me